### DISCUSSION TITLE
[Feature] Set Archive Size of Download in Administration Settings (not Account Settings)

### DISCUSSION THREAD

**Discussion by mouie** [immich-app/immich#13646] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

**Feature Request:** Set the Archive Size of Downloads at the server level (via Administration Settings), rather than user (via Account Settings).

**Rationale:**
Currently, the default is set at 4GB and needs to be updated in each User's settings. This is a helpful feature as-is, although less optimal in managing how the server is serving download requests.

I thought of this feature after observing failed downloads behind my reverse proxy and Cloudflare due to timeouts. I resolved that issue by both disabling Cloudflare's proxy, and increasing the reverse proxy's timeouts (per [this guide](https://immich.app/docs/administration/reverse-proxy/)), however I believe that being able to also reduce the Archive Size at a server level would help resolve this more efficiently.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
I've generalized this to #13651
